backend/repository/config.go

```diff
@@ -65,12 +65,43 @@ type EndpointConfig struct {
 	WorkflowType     string                    `json:"workflowType"`
 	ClientID         string                    `json:"clientID"`
 	ClientMethod     string                    `json:"clientMethod"`
-	TestFixtures     []string                  `json:"testFixtures"`
+	TestFixtures     []*TestStub                  `json:"testFixtures"`
 	Middlewares      []*codegen.MiddlewareSpec `json:"middlewares"`
 	ReqHeaderMap     map[string]string         `json:"reqHeaderMap"`
 	ResHeaderMap     map[string]string         `json:"resHeaderMap"`
 }
 
+// TestStub saves stubbed requests/responses for an endpoint test.
+type TestStub struct {
```

Comment:
Is this the same as https://github.com/uber/zanzibar/blob/master/codegen/module_system.go#L75 except the json tags? If so, why duplicate?

Reply:
yes and duplicated on purpose
1, there should be one testFixture struct in the code repo, a user passed in test case, either structured JSON (file) or go struct should have ubiquitous form
2, I'm not convinced testFixture/testStub struct we use in backend repo & runtime makes sense
-> refactor all testFixtures used in this repo with a redesigned struct that:
1, use maps instead of string list, to facilitate user edit from UI
2, use a better request / response struct, like https://github.com/uber/zanzibar/blob/master/backend/repository/handler_test.go#L44

this will be captured in another change to limit the scope